ubuntu-headless-qt/README.md

```diff
@@ -0,0 +1,996 @@
+│   ├── boot
+│   ├── dev
+│   ├── etc
+│   ├── home
+│   ├── lib -> usr/lib
+│   ├── media
+│   ├── mnt
+│   ├── opt
+│   ├── proc
+│   ├── root
+│   ├── run
+│   ├── sbin -> usr/sbin
+│   ├── snap
+│   ├── srv
+│   ├── sys
+│   ├── tmp
+│   ├── usr
+│   └── var
+└── rzpi.dtb
+```
+- Step 2: Define the boot arguments to specify the network and root file system settings:
+
+  ```shell
+  => setenv bootargs 'consoleblank=0 strict-devmem=0 ip=<device_ip>:<server_ip>::::<eth_device> root=/dev/nfs rw nfsroot=<server_ip>:</path/to/your/rootfs>,v3,tcp' 
+  ```
+
+  For example:
+  ```shell
+  => setenv bootargs 'consoleblank=0 strict-devmem=0 ip=192.168.5.30:192.168.5.86::::eth1 root=/dev/nfs rw nfsroot=192.168.5.86:/tftpboot/rzsbc/rootfs,v3,tcp'
+  ```
+
+- Step 3: Configure the boot command to load the kernel image and device tree files.
+
+  ```shell
+  => setenv bootcmd 'tftp <load_address_kernel> <path/to/kernel_image>; tftp <load_address_dtb> <path/to/device_tree_blob>; tftp <load_address_dtbo> <path/to/dtbo file>; booti <load_address_kernel> - <load_address_dtb> - <load_address_dtbo>'
+  ```
+
+  For example load `Image`, `rzpi.dtb` and `rzpi-ext-spi.dtbo` files.
+  ```shell
+  => setenv bootcmd 'tftp 0x48080000 rzsbc/Image; tftp 0x48000000 rzsbc/rzpi.dtb; tftp 0x48010000 rzsbc/overlays/rzpi-ext-spi.dtbo; booti 0x48080000 - 0x48000000 - 0x48010000'
+  ```
+
+- Step 4: Save the changes to the environment variables so they persist across reboots:
+
+  ```shell
+  => saveenv
+  ```
+
+- Step 5: Initiate the boot progress by running bootcmd:
+
+  ```shell
+  run bootcmd
+  ```
+
+  If everything is set up correctly, the images will be booted from the network.
+
+  ```
+  => run bootcmd
+  Using ethernet@11c30000 device
+  TFTP from server 192.168.5.86; our IP address is 192.168.5.30
+  Filename rzsbc/Image'.
+  Load address: 0x48080000
+  Loading: #################################################################
+          #################################################################
+          #################################################################
+          19.6 MiB/s
+  done
+  Bytes transferred = 18035200 (1133200 hex)
+  Using ethernet@11c30000 device
+  TFTP from server 192.168.5.86; our IP address is 192.168.5.30
+  Filename 'rzsbc/rzpi.dtb'.
+  Load address: 0x48000000
+  Loading: ####
+          8.6 MiB/s
+  done
+  Bytes transferred = 44855 (af37 hex)
+  Using ethernet@11c30000 device
+  TFTP from server 192.168.5.86; our IP address is 192.168.5.30
+  Filename 'rzsbc/overlays/rzpi-ext-spi.dtbo'.
+  Load address: 0x48010000
+  Loading: #
+          455.1 KiB/s
+  done
+  Bytes transferred = 932 (3a4 hex)
+  Moving Image from 0x48080000 to 0x48200000, end=493a0000
+  ## Flattened Device Tree blob at 48000000
+    Booting using the fdt blob at 0x48000000
+    Loading Device Tree to 000000007bf1a000, end 000000007bf27f36 ... OK
+
+  Starting kernel ...
+  ```
+
+### Using SSH and SCP for Remote Access and File Transfers
```

Comment:
Remove this section; I don't think Ubuntu users will find this feature interesting.